lib/callbacks.js

```diff
@@ -48,46 +40,42 @@ const once = (
   return wrap;
 };
 
-const unsafeCallback = (
-  // Extract callback function
-  // It's unsafe: may return null, allows multiple calls
-  args // array, arguments
-  // Returns: function, callback or null
-  // Hint: previous name: `common.cbUnsafe` (deprecated)
-  // Hint: another alias: `common.extractCallback` (deprecated)
-) => {
+// Extract callback function
+// It's unsafe: may return null, allows multiple calls
+//   args - array, arguments
+// Returns: function, callback or null
+// Hint: previous name: `common.cbUnsafe` (deprecated)
+// Hint: another alias: `common.extractCallback` (deprecated)
+const unsafeCallback = args => {
   const callback = last(args);
   if (typeof callback === 'function') return args.pop();
   return null;
 };
 
-const safeCallback = (
-  // Extract callback
-  args // array, arguments
-  // Returns: function, wrapped callback or common.emptiness if there is no one
-  // Hint: previous name: `cbExtract` (deprecated)
-) => {
+// Extract callback
+//   args - array, arguments
+// Returns: function, wrapped callback or common.emptiness if there is no one
```

Comment:
Call back is not wrapped in any way here, should probably be just ' callback or comm....'.